ghost/core/core/server/data/migrations/versions/5.31/2023-01-19-07-46-add-mentions-table.js

```diff
@@ -0,0 +1,23 @@
+const {addTable} = require('../../utils');
+
+// stay below 
+const mentionsTable = addTable('mentions', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    source: {type: 'string', maxlength: 2000, nullable: false},
+    source_title: {type: 'string', maxlength: 2000, nullable: true},
+    source_site_title: {type: 'string', maxlength: 2000, nullable: true},
+    source_excerpt: {type: 'string', maxlength: 2000, nullable: true},
+    source_author: {type: 'string', maxlength: 2000, nullable: true},
+    source_featured_image: {type: 'string', maxlength: 2000, nullable: true},
+    source_favicon: {type: 'string', maxlength: 2000, nullable: true},
+    target: {type: 'string', maxlength: 2000, nullable: false},
+    resource_id: {type: 'string', maxlength: 24, nullable: true},
+    resource_type: {type: 'string', maxlength: 50, nullable: true},
+    created_at: {type: 'dateTime', nullable: false},
+    payload: {type: 'text', maxlength: 1000000000, nullable: true}
```

Comment:
This seems way too big for the `payload` - we need to cap this to a reasonable size IMO something in the thousands makes more sense to me - but we should check with Nick & Peter